scripts/index.js

```diff
@@ -0,0 +1,74 @@
+function getRandomInt(min, max) {
+	min = Math.floor(min);
+	max = Math.ceil(max);
+	return Math.floor(Math.random() * (max - min +1)) + min;
+}
+
+
+let diceOne = {
+	value: 1,
+	roll : function(){
+		this.value = getRandomInt(1, 6);
+		return this.value;
+	}
+};
+let diceTwo = {
+	value: 1,
+	roll : function(){
+		this.value = getRandomInt(1, 6);
+		return this.value;
+	}
+};
+// console.log(diceOne.value);
+//
+// diceOne.roll();
+// console.log(diceOne.value);
+//
+// diceOne.roll();
+// console.log(diceOne.value);
+//
+// diceOne.roll();
+// console.log(diceOne.value);
+//
+
+
+let button = document.body.querySelector("#roll-dice");
+
+
+
+const loadDice = event =>{
+	if(event) {
+		event.preventDefault;
+	}
+	
+
+	let containerOne = document.body.querySelector("#container-1");
```

Comment:
Try to separate the whole logic responsible for changing the UI into it's own function. This is good for two reasons

0. It makes the code more readable
0. It will enable you to abstract and reuse the functions for changing the UI in other ways, thus increasing flexibility

The principle of making your code more flexible by making sure that each function (or class) is responsible for as little as possibly is called the _Single Responsibility Principle_. You can read about it here:
https://codepen.io/allanpope/post/single-responsibility-principle#single-responsibility-1